src/pages/index.js

```diff
@@ -245,3 +373,32 @@ addNewCardButton.addEventListener("click", () => {
 //   popup.classList.remove("modal_opened");
 //   document.removeEventListener("keydown", handleEscapePress);
 // }
+
+//darken effect
+
+document
+  .querySelector(".profile__avatar")
+  .addEventListener("mouseenter", function () {
+    document.querySelector(".profile__image").style.filter = "brightness(0.2)";
+  });
+
+document
+  .querySelector(".profile__avatar")
+  .addEventListener("mouseleave", function () {
+    document.querySelector(".profile__image").style.filter = "brightness(1)";
+  });
+
+function handleDeleteCard(cardId, card) {
+  console.log(cardId);
```

Comment:
You should also notify the user about loading process by changing the submit button text